_includes/navbar.html

```diff
@@ -51,7 +51,11 @@
           {%- assign url = html_page.url | relative_url -%}
         {%- endif -%}
 
-        <a class="nav-item nav-link {{ active_status }}" href="{{ url }}">{{ html_page.title }}</a>
+        {%- if html_page.open_new_tab -%}
+          <a class="nav-item nav-link {{ active_status }}" href="{{ url }}" target="_blank">{{ html_page.title }}</a>
+        {%- else -%}
+          <a class="nav-item nav-link {{ active_status }}" href="{{ url }}">{{ html_page.title }}</a>
+        {%- endif -%}
```

Comment:
An inline if statement would be cleaner.

```suggestion
          <a class="nav-item nav-link {{ active_status }}" href="{{ url }}" {%- if html_page.open_new_tab -%} target="_blank" {%- endif -%}>{{ html_page.title }}</a>
```